### DISCUSSION TITLE
Run Nuxt script in existing terminal session

### DISCUSSION THREAD

**Discussion by edimitchel** [nuxtrdev/nuxtr-vscode#7] [Ideas]
Running a project script from the command panel or from the sidenav is great but in my opinion, I'm not confortable by always opening a new terminal especially when it's a script which is terminating (no watch mode): it's open a new terminal and then ends by showing a stale terminal window which waits for key press to close it.

I don't know how to do, but I think we could permits to reuse an already opened idle terminal and run the script into. That would avoid a new terminal window and give back control at the same position than before when script is ended or stopped.

> Disclaimer: I'm may be wrong as I didn't try the extension yet, so let close this discussion if my proposal is out of scope.

**Comment by 0kaa** [comment]
Hi @edimitchel 
Thanks for your idea, we will investigate on this feature to show output only when the script has no watch mode
